### PR TITLE
Add CISCO AMP integration

### PR DESCRIPTION
**Pull request by Spivaka**
fixes https://github.com/demisto/server/issues/1526

### CODE REVIEW

**Comment by Spivaka** [comment]
@aviadl please take a look.
To my understanding, AMP and SourceFire has the same API (from this https://api-docs.amp.sourcefire.com/), so I think this handles both.

File: Integrations/integration-AMP.yml [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+        var query = '';
+        if (args) {
+            var keys = Object.keys(args);
+            if (keys.length > 0) {
+                query = '?';
+                for (var i = 0; i<keys.length; i++) {
+                    if (i !== 0) {
+                        query += '&';
+                    }
+                    query += encodeURIComponent(fixBracketsArgs(command, keys[i])) + '=' + encodeURIComponent(args[keys[i]]);
```
Comment (aviadl) [inline Integrations/integration-AMP.yml]: Do not encode key, need to encode only values

File: Integrations/integration-AMP.yml [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+    var encodeBody = function(args) {
+        if (args) {
+            return JSON.stringify(args);
+        }
+        return undefined;
+    }
+
+    switch (command) {
+        case 'test-module':
+            return 'ok';
```
Comment (aviadl) [inline Integrations/integration-AMP.yml]: Missing test

**Comment by rishibhargava** [comment]
Sourcefire was the company that made an endpoint product called amp and then the original product which was IPS. Look for firesight API.

Rishi

________________________________

**Pull request merged**